tests/Io/FileSystemTest.php

```
@@ -42,6 +42,130 @@ public function tearDown()
         $this->exec('rm -rf ' . $this->tmp);
     }
 
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error getting real path of invalid_path, check that the path exists
+     */
+    public function testGetRealPathWithInvalidPath()
+    {
+        FileSystem::getRealPath('invalid_path');
+    }
+
+    public function testGetRealPath()
+    {
+        $filePath = __DIR__ . '/../../composer.json';
+        $expectedRealPath = realpath($filePath);
+
+        $this->assertSame($expectedRealPath, FileSystem::getRealPath($filePath));
+    }
+
+    public function testWriteWithAppendFlag()
+    {
+        $this->assertSame(4, FileSystem::write('./temp_file', 'data', true));
+    }
+
+    public function testWriteWithLockFlag()
+    {
+        $this->assertSame(0, FileSystem::write('./temp_lock_file', '', false, true));
+    }
+
+    public function testReadWithMaxLength()
+    {
+        FileSystem::write('./temp_lock_file', 'data');
+
+        $this->assertSame('dat', FileSystem::read('./temp_lock_file', 0, 3));
+    }
+
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error copying ./temp_lock_file to /temp_lock_file
+     */
+    public function testCopyShouldReturnIOException()
```

Can you please rename those methods to `*ThrowIOException` instead of `Return`?